browserstack-karma.js

```diff
@@ -31,27 +31,31 @@ module.exports = {
   },
 
   // Older mainstream
+  /* https://github.com/snabbdom/snabbdom/issues/468
```

Comment:
@nojvek I agree. Here are comments linking to specific issues. Details on those failures could be added to those issues later (or just solved).